openthread_border_router/Dockerfile

```diff
@@ -63,25 +63,30 @@ RUN \
         -DOTBR_FEATURE_FLAGS=ON \
         -DOTBR_DNSSD_DISCOVERY_PROXY=ON \
         -DOTBR_SRP_ADVERTISING_PROXY=ON \
-        -DOTBR_INFRA_IF_NAME=eth0 \
         -DOTBR_MDNS=mDNSResponder \
         -DOTBR_VERSION= \
         -DOT_PACKAGE_VERSION= \
         -DOTBR_DBUS=OFF \
-        -DOT_POSIX_CONFIG_RCP_BUS=UART \
+        -DOT_POSIX_RCP_BUS_UART=ON \
         -DOT_LINK_RAW=1 \
         -DOTBR_VENDOR_NAME="Home Assistant" \
         -DOTBR_PRODUCT_NAME="OpenThread Border Router" \
         -DOTBR_WEB=ON \
         -DOTBR_BORDER_ROUTING=ON \
         -DOTBR_REST=ON \
         -DOTBR_BACKBONE_ROUTER=ON \
+        -DOTBR_TREL=ON \
+        -DOTBR_NAT64=ON \
+        -DOT_POSIX_NAT64_CIDR="192.168.255.0/24" \
```

Comment:
should this be configurable if users running on the 10.x private subnet ? Or is that unrelated ?

Reply:
It should not matter if you use a 10.x.x.x private subnet. It could be possible problematic when using 192.168.255.x in a network, maybe :thinking: FWIW, the value is just the default value of OTBR. If I don't configure it explicitly here, the build system would use the same value implicitly.

It is possible to change the setting at runtime, see https://github.com/openthread/openthread/blob/main/src/cli/README.md#nat64-cidr. 

However, I am not sure if it is necessary at this point. I'd suggest that we leave that static default currently and see if someone runs into problems.